src/arcade_collection/output/parse_growth_file.py

```diff
@@ -0,0 +1,143 @@
+import json
+from os import path
+import tarfile
+from typing import List, Union
+
+import ntpath
```

Comment:
No need to use `ntpath`, just use `os.path` instead, which will automatically pick the right module for the OS the code is running on.